azurerm/internal/services/cdn/cdn_endpoint_resource.go

```diff
@@ -202,9 +187,40 @@ func resourceArmCdnEndpoint() *schema.Resource {
 			"tags": tags.Schema(),
 		},
 	}
+
+	return &schema.Resource{
+		Create: resourceArmCdnEndpointCreate,
+		Read:   resourceArmCdnEndpointRead,
+		Update: resourceArmCdnEndpointUpdate,
+		Delete: resourceArmCdnEndpointDelete,
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(30 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(30 * time.Minute),
+			Delete: schema.DefaultTimeout(30 * time.Minute),
+		},
+
+		Importer: azSchema.ValidateResourceIDPriorToImport(func(id string) error {
+			_, err := parse.CdnEndpointID(id)
+			return err
+		}),
+
+		Schema: tmpResource.Schema,
```

Comment:
the schema needs to remain here - and be entirely duplicated for the migration; else if these differ in the future the original state migration schema is no longer valid - so unfortunately these cannot be reused and must be copy/pasted as a point-in-time reference to the current schema